components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -120,6 +122,27 @@ public static DefaultAuthenticationRequestHandler getInstance() {
         return instance;
     }
 
+    /**
+     * This method returns the comma separated list of federated authenticator names bounded to the
+     * federated_tokens property in the authentication context.
+     *
+     * @param context Authentication context containing the federated_tokens property.
+     * @return Comma separated list of the federated authenticator names.
+     */
+    public static String getFederatedAuthenticatorName(AuthenticationContext context) {
```

Comment:
make this private

Reply:
since this util is specific to the federated tokens, lets pass in the federated tokens list here

Reply:
also lets move this util method to the end of the class

Reply:
Fixed a0f882011c467a8f0c781821bfcd6ad8f03da0c5